src/optimum/nvidia/models/llama.py

```diff
@@ -177,28 +163,6 @@ def allocate_model(
             == QuantMode(0),  # Disable if quantization for now as it remove one scaling factor
         )
 
-    @classmethod
-    def from_safetensors(
```

Comment:
This is abstracted away in `weights/base.py` since it is a shared method between several `XxxWeightsAdapter` subclasses.